abel/transform.py

```diff
@@ -101,7 +101,10 @@ class Transform(object):
 
     symmetry_axis : None, int or tuple
         Symmetrize the image about the numpy axis
-        0 (vertical), 1 (horizontal), (0, 1) (both axes)
+        0 (vertical), 1 (horizontal), (0, 1) (both axes). Note that the
+        abel transform is always performed around the y-axis.
```

Comment:
Actually, your original statement of [`z-axis`](https://github.com/PyAbel/PyAbel/issues/285#issuecomment-619503209) is correct, according to our wine-glass figure:
![image](https://user-images.githubusercontent.com/10932229/80302498-c88e7680-87ed-11ea-9bfd-bd998edb91b5.png)

Reply:
It's probably better to call it “vertical” for clarity and consistency.

Reply:
Yes I hadn't noticed there is an ambiguity in what I mean with the y-axis. I meant the vertical axis, which contradicts the image. So changing 'y-axis' to 'vertical axis' would be better.